src/ontology/oeo.omn

```diff
@@ -80,3 +83,6 @@ AnnotationProperty: dct:title
 Datatype: rdf:PlainLiteral
 
     
+Datatype: xsd:string
```

Comment:
Also xsd;string is not used in oeo.omn. In general, our restructuring should not change oeo.omn.

Reply:
```suggestion

```